stock_picking_report_custom_description/models/sale_order_line.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2023 Tecnativa - Carolina Fernandez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    def write(self, vals):
+        res = super().write(vals)
+        if vals.get("name"):
+            self.env["stock.move"].search([("sale_line_id", "=", self.id)]).name = vals[
```

Comment:
This is not correct.

```suggestion
            self.env["stock.move"].search([("sale_line_id", "in", self.ids)]).update({"name": vals["name"]})
```

Reply:
OK about the `in` operator, but you don't need `update`, as dot notation serves also for writing in a multi-record.

Reply:
Thanks @rousseldenis :). I just changed. As @pedrobaeza said, we don't need the `update` anyway. I'll check v16, as this is a potential bug.

Reply:
Yes of course